.pre-commit-config.yaml

```diff
@@ -42,3 +42,10 @@ repos:
         language: system
         entry: ./check_readme.sh
         pass_filenames: false
+      - id: gha-sha-convert
+        name: GitHub Actions SHA Converter
+        description: Convert GitHub Actions to use SHA-pinned versions
+        entry: python3 gha-sha-convert/gha_sha_convert.py
+        language: system
+        files: ^\.github/(workflows|actions)/.*\.(yml|yaml)$
+        pass_filenames: true
```

Comment:
it feels ugly but I think it would be better if we self-reference this repo here with a release tag so we avoid duplicating configs and we ensure it's really working as expected

which means this won't run the hook if changed in a branch but maybe it's a feature